docs/hooks/hook_civicrm_alterMailer.md

```diff
@@ -61,8 +61,3 @@ Introduced in CiviCRM v4.4.
       }
     }
 
-[Jon Goldberg](https://wiki.civicrm.org/confluence/display/~palantejon), good catch spotting a
```

Comment:
@seanmadsen this seemed kinda odd i think it must have been a comment probably by Tim on the wiki doc